### PR TITLE
Docs(ES-71): Update ReadMe

### PR DESCRIPTION
**Pull request by 365Bit**
Updates the ReadMe file to the new format for DeMAF plugins.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,11 +1,26 @@
...
+java -jar target/visualization-service-0.1.0-SNAPSHOT.jar
 ```
 
-## Run
-```shell
-java -jar target/visualization-service-0.1.0-SNAPSHOT.jar
-```
\ No newline at end of file
+## Visualization Service-Specific Configurations
+This plugin has a few specialties compared to other DeMAF transformation plugins:
```
Comment (eliasmueller) [inline README.md:20]: its not really a "transformation" plugin

File: README.md [hunk 2]
```diff
@@ -1,11 +1,26 @@
...
+1. Automatic execution: As long as the option `visualize=false` is not specified during the transformation, this plugin is automatically called to visualize the model after a successful transformation into an EDMM model.
```
Comment (heidrifx) [inline README.md:21]: maybe add an example on how to use the *options* flag?
i.e. `transform -f [file] -t [technology] -c [command] -option "visualize=false"

what about the other available options? (i.e. dpi, width, height, etc.)?

File: README.md (hunk 1 above)
Comment (365Bit) [reply README.md]: Okay, it has been removed.

File: README.md (hunk 2 above)
Comment (365Bit) [reply README.md:38]: A usage section has been added.

**Review by eliasmueller — APPROVED** [review]

**Pull request merged**